package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@nypl/dgx-header-component",
-  "version": "2.6.0",
+  "version": "2.7.1-r16",
   "description": "NYPL Header as a React component.",
-  "main": "dist/components/Header/Header.js",
+  "main": "dist/index.min.js",
   "scripts": {
-    "start": "NODE_ENV=development node server.js",
-    "babel-build": "./node_modules/.bin/babel src -d dist && cp -r src/styles/ dist/styles",
+    "start": "NODE_ENV=production node server.js",
+    "start:dev": "NODE_ENV=development webpack-dev-server --open",
     "dist": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.js && cp -r src/styles/ dist/styles",
```

Comment:
Replacing `babel-build` with `dist`. This removed a lot of unminified files in `/dist` with a single `index.min.js` file.